.changeset/flat-badgers-kiss.md

```diff
@@ -0,0 +1,9 @@
+---
+"no-bundle-import": patch
+"node-app-pages": patch
+"pages-workerjs-app": patch
+"pages-workerjs-directory": patch
+"remix-pages-app": patch
```

Comment:
Do we usually changeset the fixtures? I've never seen us do that before.

Reply:
Yeah ... good question. Happy to go either way.

Reply:
Removing